src/foremast/runner.py

```diff
@@ -134,7 +134,12 @@ def create_archaius(self):
     def create_s3app(self):
         """Create S3 infra for s3 applications"""
         utils.banner("Creating S3 App Infrastructure")
-        s3obj = s3.S3Apps(app=self.app, env=self.env, region=self.region, prop_path=self.json_path)
+        primary_region = self.configs['pipeline']['primary_region']
```

Comment:
can we call this default_region? We can then use this setting as a default similar how AWS CLI works :)

Reply:
we just call it "primary_region" in the configs, so I stuck with that terminology here. It is a value you can override in pipeline.json and originally was used for the multi-region DNS stuff.

Reply:
Got it. What we called it elsewhere so lets remain consistent. 👍